composer.json

```diff
@@ -12,20 +12,21 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.9",
-        "symfony-cmf/routing-auto": "~1.0",
-        "symfony-cmf/routing-bundle": "~1.2",
+        "symfony-cmf/routing-auto": "~1.1",
+        "symfony-cmf/routing-bundle": "~1.2,>=1.2.0",
         "symfony-cmf/core-bundle": "~1.2",
         "aferrandini/urlizer": "1.0.*",
         "symfony/config": "~2.2",
         "jms/metadata": "1.5.*"
     },
     "require-dev": {
-        "symfony-cmf/testing": "~1.1",
+        "symfony-cmf/testing": "dev-fix_container_creation as 1.2.7",
```

Comment:
this needs to be reverted to ``"symfony-cmf/testing": "~1.2,>=1.2.7"``, once symfony-cmf/Testing#109 is merged and tagged as 1.2.7